home/ivp-planning/advanced-tools/local-library.md

```diff
@@ -0,0 +1,56 @@
+# Local Library
+
+You can import your own files/objects from your PC into iVP by using the "Import" option, found under the "File" menu at the top of the application.  
```

Comment:
```suggestion
You can import your own files/objects from your PC into Planning by using the "Import" option, found under the "File" menu at the top of the application.  
```

General note: Don't refer to iVP Planning as iVP, iVP Stage is also iVP. If you want to make it short, just say "Planning".